src/librustc_mir/borrow_check/type_check/mod.rs

```diff
@@ -1468,7 +1469,7 @@ impl<'a, 'tcx> TypeChecker<'a, 'tcx> {
                 self.check_rvalue(body, rv, location);
                 if !self.tcx().features().unsized_locals {
                     let trait_ref = ty::TraitRef {
-                        def_id: tcx.lang_items().sized_trait().unwrap(),
+                        def_id: tcx.require_lang_item(SizedTraitLangItem, None),
```

Comment:
Since we have a `stmt`, I think we can pass a meaningful span here instead of `None`.

Reply:
In fact we can probably do better and use `self.last_span`, also most other places in this file.
(Found by inspecting what the `span_mirbug!` macro does.)